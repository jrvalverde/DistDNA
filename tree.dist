	Chimpance 	Gorila    	Macaco jap	Titi      	Orangutan 	H sapiens 	Aborigen a	H sapiens 	H sapiens 	Geospiza m	Caretta ca	tortuga de	Caballo ar	Cerdo     
Chimpance 	  0.00														
Gorila    	0.1660	  0.00													
Macaco jap	0.3975	0.3945	  0.00												
Titi      	0.4250	0.4658	0.3540	  0.00											
Orangutan 	0.2461	0.2408	0.3464	0.4000	  0.00										
H sapiens 	0.1434	0.2416	0.4498	0.4313	0.2461	  0.00									
Aborigen a	0.1473	0.2082	0.3633	0.4250	0.2408	0.1929	  0.00								
H sapiens 	0.1434	0.2007	0.3599	0.4375	0.2356	0.1958	0.0087	  0.00							
H sapiens 	0.1434	0.2007	0.3599	0.4313	0.2408	0.2018	0.0087	0.0058	  0.00						
Geospiza m	0.6066	0.6260	0.6210	0.6186	0.6341	0.6311	0.6148	0.6311	0.6230	  0.00					
Caretta ca	0.7086	0.7600	0.6606	0.6732	0.7143	0.7143	0.7143	0.7200	0.7143	0.5932	  0.00				
tortuga de	0.7151	0.7442	0.6358	0.6358	0.7151	0.7209	0.7209	0.7267	0.7267	0.6000	0.1992	  0.00			
Caballo ar	0.6905	0.7024	0.6772	0.6531	0.6786	0.6905	0.6964	0.6964	0.7083	0.6134	0.4853	0.4589	  0.00		
Cerdo     	0.7087	0.6875	0.6615	0.6639	0.6562	0.7008	0.6929	0.6850	0.7008	0.6068	0.4603	0.4567	0.2756	  0.00	
