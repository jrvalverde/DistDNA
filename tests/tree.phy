    14    423
Chimpancé  ---------- ---------- ---------- ---------- ---------- 
Gorila     ---------- ---------- ---------- ---------- ---------- 
Macaco_jap ---------- ---------- ---------- ---------- ---------- 
Tití       ---------- ---------- ---------- ---------- ---------- 
Orangután_ ---------- ---------- ---------- ---------- ---------- 
H.sapiens_ ---------- ---------- ---------- ---------- ---------- 
Aborigen_a ---------- ---------- ---------- ---------- ---------- 
H.sapiens_ ---------- ---------- ---------- ---------- ---------- 
H.sapiens_ ---------- ---------- ---------- ---------- ---------- 
Geospiza_m ---------- ---------- ---------- ---------- ---------- 
Caretta_ca TGATACCTGC CGAATTGATG AAACTGAGCC TACGTTCAAA -ATGATTGGC 
tortuga_de TGATACCTGC CGATTCAGTG AAACTGGACT TACGTTTAAA TATGATTGGC 
Caballo_ár ---------- ---------- -----GGACT TAAATTGAAC GTTATCCCTC 
Cerdo      ---------- ---------- ---------- ---------- ---------- 

           ---------- ----TTTTCG TCTGGGGGGT GTGCACGCGA TAGCATTGCG 
           ---------- ----TTTTCG TC-GGGGGGT GTGCACGCGA TAGCATTGCG 
           ---------- ---------- -------GGT CTGCACGCAA CCCCATTGCA 
           ---------- ---------- -------GGT ATGTACTCAT CAGCATTGCC 
           ---------- -----TTTTT TCGGGGGGGG ATGCACGCGA TAGCATCGCG 
           ---------- ----TTTTCG TCTGGGGGGT GTGCACGCGA TAGCATTGCG 
           ---------- ----TTTTCG TCTGGGGGGT GTGCACGCGA TAGCATTGCG 
           ---------- ----TTTTCG TCTGGGGGGT GTGCACGCGA TAGCATTGCG 
           ---------- ----TTTTCG TCTGGGGGGT GTGCACGCGA TAGCATTGCG 
           ---------- ---------- ---------- ---------- ---------- 
           CGTGCAGAAT ACTTAATGGT ATTATTTAAT TAATGCTTTT AGGACATATA 
           CGTGCAAACT GATTAATGGT ATTATTAAGT TAATGCTTAT AAGACATAGA 
           CGCATAAGCA ACCATAAGGT GTTATTCAGT TCATGGTAGC AGGACATAGG 
           ---------- ---------- ---------- ---------- ---------- 

           AAACGCTGGC CCCGGAGCAC --CCTATGTC GCA-GTATCT GTCTTTGATT 
           AAACGCTGGA ACCGGAGCAC --CACATGTC GCA-GTATCT GTCTTTGATT 
           GTATGCTGAC TCCC-ACCAC ATCTCGTCCT GAATGCGCCT GTCTTTGATT 
           G---GGAGGT TCCA-ACCA- ---TCATCCT GTA-GCCCCT GACTGTGTTT 
           GGCCGCTGGA ACCGGAGCAC --CCTATGTC GCA-GGATCT GTCTTTGATT 
           AGACGCTGGA GCCGGAGCAC --CCTATGTC GCA-GTATCT GTCTTTGATT 
           AGACGCTGGA GCCGGAGCAC --CCTATGTC GCA-GTATCT GTCTTTGATT 
           AGACGCTGGA GCCGGAGCAC --CCTATGTC GCA-GTATCT GTCTTTGATT 
           AGACGCTGGA GCCGGAGCAC --CCTATGTC GCA-GTATCT GTCTTTGATT 
           --------CT ACATTTCCTG CTAACATAAA ACA-ACAACT AACCATCATC 
           TTTTTATAAA -AACTCACAA CAGTTATT-T ACAAGCTAAA ACCCATTAC- 
           ATTTCACAAT TAAACCTAAA CAATGATC-T ACAACCTAA- -CTCATTATT 
           AAACAAGTGC ACCTGTGCAC CTGTGCACCT GCCCGCGCAG -C-AAGCAAG 
           -------TAC ACGTGCGTAC ACGTGCG--T ACACGCGCAT -ATAAGCAGG 

           CCTACCCCAT TACATTATTT ATCGCACCT- A-CGTTCAAT ATTACGGCCC 
           CCTGCCCCAT ACCATTATTT ATCGCACCT- A-CGTTCAAT ATTACAGCCG 
           CCTAGTACAT GCAGTTGTTG ATCGCACCT- A-CGTTCAAT ATTCTAGCTC 
           TGGCAGACTT TATGCTATC- ATCGCGCCT- A-TATTGAAT GTCTTGGGCC 
           CCTACCTCAT GCCATTATTA ATCGCGCCT- AATATCCAAT ATCCTAGCCC 
           CCTGCCCCAT TCCATTATTT ATCGCACCT- A-CGTTCAAT ATTACAGGCG 
           CCTGCCTCAT TCTATTATTT ATCGCACCT- A-CGTTCAAT ATTACAGGCG 
           CCTGCCTCAT CCCATTATTT ATCGCACCT- A-CGTTCAAT ATTACAGGCG 
           CCTGCCTCAT CCTATTATTT ATCGCACCT- A-CGTTCAAT ATTACAGGCG 
           ACCACCATAT TTTCCTCCCC TTTTCCCCT- ACCCCACACA AAACAGCCCT 
           AACCATACTT T--TTAGTTA AACCCCCCC- -ACCCCCAT- AAACT-AACA 
           AACTGTACTT T--TTAGCTA AACCCCCCT- -ACCCCCGTT AAAGTCAACA 
           TAATATAGCT TTCTTAATCA AACCCCCCT- ACCCCCCATT AAACT----- 
           TAA-ATTATT AGCTCATTCA AACCCCCCTT ACCCCCCATT AAACT----- 

           AGCATACC-T ATTGAAGCGT GTTAATTAAT TAATGCTTGT AGGACATAAC 
           AGCGCAGCAT GCTTTATGAT GCTAATTAAT TCATGCTTGT TGGACATAAA 
           CACGCAAACT TTAGCAAGGT GTTATTTAAT TCATGCTTGT AGGACATACT 
           CCATCCCACC C-ACTAAGGT GTAATTTA-T TAATGGTCAC AGGACATAAT 
           CACCCTCAGT GTTTGAAGCT GCTATTTAAT TTATGCT-AG AGGACATAAA 
           AGCATACT-T ACTAAAGTGT GTTAATTAAT TAATGCTTGT AGGACATAAT 
           AACATACT-T ACTAAAGTGT GTTAATTAAT TAATGCTTGT AGGACATAAT 
           AACATACT-T ACTAAAGTGT GTTAATTAAT TAATGCTTGT AGGACATAAT 
           AACATACC-T ACTAAAGTGT GTTAATTAAT TAATGCTTGT AGGACATAAT 
           TACAAAAACC AAACAAAAAT ACAAACCA-- TAACACCA-- ---------- 
           TTATGCCCGA ATAGCTATTC ACTTCTCGTC AAACCCCTAA ATCCGAGACT 
           CCAGCCCGCT ATAGCCATTT ACTTCTCGCC AAACCCCTAA ATCCGAGA-- 
           CCACATATGT ACATTCAACA CAATCTTGCC AAACCCCAAA AACAA----- 
           TATGCTCTAC ACACCCTATA ACGCCTTGCC AAACCCCAAA AACAAAGCA- 

           AACAGCAGCA AAATGTTCAC ACAACCGCTT TCCACACCAA CA-CATAACA 
           ACAACCAGGT GGACGTGAAC ACAACCACCC TTCACACCAA AAACACAACA 
           AATAACCATC CTGGTCGGTA CCACTCACAC CACACCATTA AC-CACAACC 
           AA-------- ---------- ---------- ---------- ---------- 
           ATTACCAAAA AAA------- ---------- ---------- ---------- 
           AATAACGACT AAATGTCTGC ACAGCTGCTT TCCACACAGA CATCATAACA 
           AATAACAATT GAATGTCTGC ACAGCTGCTT TCCACACAGA CATCATAACA 
           AATAACAATT GAATGTCTGC ACAGCCGCTT TCCACACAGA CATCATAACA 
           AATAACAATT GAATGTCTGC ACAGCCGCTT TCCACACAGA CATCATAACA 
           ---------- ---------- ---------- ---------- ---------- 
           AAC------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 

           AAAATTCCCA C---AAACCC CCC--CCTCC CCCC------ ---------- 
           AAAAATCC-- -----AACCC CCC--CCCCC CCGCACTAGA AAACAGCCC- 
           GTATCTTATC A---AACCCC CCC--ACCCC CATCTCCGAC CTTCATCCA- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           AAAAATTTCC A---CCAAAC CCC--CTTTC CTCCCCCGCT TCTGGCCAC- 
           AAAAATTTCC ACCAAACCCC CCC--TCCCC CCGCTTCTGG CCACAGCACT 
           AAAAATTTCC ACCAAACCCC CCCCCTCCCC CCGCTTCTGG CCACAGCACT 
           AAAAATTTCC ACCAAACCCC CCC--TCCCC CCGCTTCTGG CCACAGCACT 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 

           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           --AACCCACT CTTGCCAAAC CCCAAAAACA AA-------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           --AGCACTTA AACACATCTC TGCCAAACCC CAAAAACAAA GAACCCTAAC 
           TAAACACATC TCTGCCAAAC CCCAAAAACA AAGAACCCTA ACACCAGCCT 
           TAAACACATC TCTGCCAAAC CCCAAAAACA AAGAACCCTA ACACCAGCCT 
           TAAACACATC TCTGCCAAAC CCCAAAAACA AAGAACCCTA ACACCAGCCT 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 
           ---------- ---------- ---------- ---------- ---------- 

           ---------- ---------- ---
           ---------- ---------- ---
           ---------- ---------- ---
           ---------- ---------- ---
           ---------- ---------- ---
           ACCAGCCTAA CCAGACTTCA AAT
           AACCAGATTT CAAAT----- ---
           AACCAGATTT CAAAT----- ---
           AACCAGATTT CAAAT----- ---
           ---------- ---------- ---
           ---------- ---------- ---
           ---------- ---------- ---
           ---------- ---------- ---
           ---------- ---------- ---
