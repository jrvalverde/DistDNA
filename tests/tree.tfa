>DL;Chimpancé

--------------------------------------------------
--------------TTTTCGTCTGGGGGGTGTGCACGCGATAGCATTGCG
AAACGCTGGCCCCGGAGCAC--CCTATGTCGCA-GTATCTGTCTTTGATT
CCTACCCCATTACATTATTTATCGCACCT-A-CGTTCAATATTACGGCCC
AGCATACC-TATTGAAGCGTGTTAATTAATTAATGCTTGTAGGACATAAC
AACAGCAGCAAAATGTTCACACAACCGCTTTCCACACCAACA-CATAACA
AAAATTCCCAC---AAACCCCCC--CCTCCCCCC----------------
--------------------------------------------------
-----------------------
>DL;Gorila

--------------------------------------------------
--------------TTTTCGTC-GGGGGGTGTGCACGCGATAGCATTGCG
AAACGCTGGAACCGGAGCAC--CACATGTCGCA-GTATCTGTCTTTGATT
CCTGCCCCATACCATTATTTATCGCACCT-A-CGTTCAATATTACAGCCG
AGCGCAGCATGCTTTATGATGCTAATTAATTCATGCTTGTTGGACATAAA
ACAACCAGGTGGACGTGAACACAACCACCCTTCACACCAAAAACACAACA
AAAAATCC-------AACCCCCC--CCCCCCCGCACTAGAAAACAGCCC-
--------------------------------------------------
-----------------------
>DL;Macaco_japonés

--------------------------------------------------
---------------------------GGTCTGCACGCAACCCCATTGCA
GTATGCTGACTCCC-ACCACATCTCGTCCTGAATGCGCCTGTCTTTGATT
CCTAGTACATGCAGTTGTTGATCGCACCT-A-CGTTCAATATTCTAGCTC
CACGCAAACTTTAGCAAGGTGTTATTTAATTCATGCTTGTAGGACATACT
AATAACCATCCTGGTCGGTACCACTCACACCACACCATTAAC-CACAACC
GTATCTTATCA---AACCCCCCC--ACCCCCATCTCCGACCTTCATCCA-
--AACCCACTCTTGCCAAACCCCAAAAACAAA------------------
-----------------------
>DL;Tití

--------------------------------------------------
---------------------------GGTATGTACTCATCAGCATTGCC
G---GGAGGTTCCA-ACCA----TCATCCTGTA-GCCCCTGACTGTGTTT
TGGCAGACTTTATGCTATC-ATCGCGCCT-A-TATTGAATGTCTTGGGCC
CCATCCCACCC-ACTAAGGTGTAATTTA-TTAATGGTCACAGGACATAAT
AA------------------------------------------------
--------------------------------------------------
--------------------------------------------------
-----------------------
>DL;Orangután_de_Sumatra

--------------------------------------------------
---------------TTTTTTCGGGGGGGGATGCACGCGATAGCATCGCG
GGCCGCTGGAACCGGAGCAC--CCTATGTCGCA-GGATCTGTCTTTGATT
CCTACCTCATGCCATTATTAATCGCGCCT-AATATCCAATATCCTAGCCC
CACCCTCAGTGTTTGAAGCTGCTATTTAATTTATGCT-AGAGGACATAAA
ATTACCAAAAAAA-------------------------------------
--------------------------------------------------
--------------------------------------------------
-----------------------
>DL;H.sapiens_neanderthalensis

--------------------------------------------------
--------------TTTTCGTCTGGGGGGTGTGCACGCGATAGCATTGCG
AGACGCTGGAGCCGGAGCAC--CCTATGTCGCA-GTATCTGTCTTTGATT
CCTGCCCCATTCCATTATTTATCGCACCT-A-CGTTCAATATTACAGGCG
AGCATACT-TACTAAAGTGTGTTAATTAATTAATGCTTGTAGGACATAAT
AATAACGACTAAATGTCTGCACAGCTGCTTTCCACACAGACATCATAACA
AAAAATTTCCA---CCAAACCCC--CTTTCCTCCCCCGCTTCTGGCCAC-
--AGCACTTAAACACATCTCTGCCAAACCCCAAAAACAAAGAACCCTAAC
ACCAGCCTAACCAGACTTCAAAT
>DL;Aborigen_australiano

--------------------------------------------------
--------------TTTTCGTCTGGGGGGTGTGCACGCGATAGCATTGCG
AGACGCTGGAGCCGGAGCAC--CCTATGTCGCA-GTATCTGTCTTTGATT
CCTGCCTCATTCTATTATTTATCGCACCT-A-CGTTCAATATTACAGGCG
AACATACT-TACTAAAGTGTGTTAATTAATTAATGCTTGTAGGACATAAT
AATAACAATTGAATGTCTGCACAGCTGCTTTCCACACAGACATCATAACA
AAAAATTTCCACCAAACCCCCCC--TCCCCCCGCTTCTGGCCACAGCACT
TAAACACATCTCTGCCAAACCCCAAAAACAAAGAACCCTAACACCAGCCT
AACCAGATTTCAAAT--------
>DL;H.sapiens_Indonesio

--------------------------------------------------
--------------TTTTCGTCTGGGGGGTGTGCACGCGATAGCATTGCG
AGACGCTGGAGCCGGAGCAC--CCTATGTCGCA-GTATCTGTCTTTGATT
CCTGCCTCATCCCATTATTTATCGCACCT-A-CGTTCAATATTACAGGCG
AACATACT-TACTAAAGTGTGTTAATTAATTAATGCTTGTAGGACATAAT
AATAACAATTGAATGTCTGCACAGCCGCTTTCCACACAGACATCATAACA
AAAAATTTCCACCAAACCCCCCCCCTCCCCCCGCTTCTGGCCACAGCACT
TAAACACATCTCTGCCAAACCCCAAAAACAAAGAACCCTAACACCAGCCT
AACCAGATTTCAAAT--------
>DL;H.sapiens_sapiens

--------------------------------------------------
--------------TTTTCGTCTGGGGGGTGTGCACGCGATAGCATTGCG
AGACGCTGGAGCCGGAGCAC--CCTATGTCGCA-GTATCTGTCTTTGATT
CCTGCCTCATCCTATTATTTATCGCACCT-A-CGTTCAATATTACAGGCG
AACATACC-TACTAAAGTGTGTTAATTAATTAATGCTTGTAGGACATAAT
AATAACAATTGAATGTCTGCACAGCCGCTTTCCACACAGACATCATAACA
AAAAATTTCCACCAAACCCCCCC--TCCCCCCGCTTCTGGCCACAGCACT
TAAACACATCTCTGCCAAACCCCAAAAACAAAGAACCCTAACACCAGCCT
AACCAGATTTCAAAT--------
>DL;Geospiza_magnirostris-ave

--------------------------------------------------
--------------------------------------------------
--------CTACATTTCCTGCTAACATAAAACA-ACAACTAACCATCATC
ACCACCATATTTTCCTCCCCTTTTCCCCT-ACCCCACACAAAACAGCCCT
TACAAAAACCAAACAAAAATACAAACCA--TAACACCA------------
--------------------------------------------------
--------------------------------------------------
--------------------------------------------------
-----------------------
>DL;Caretta_caretta-tortuga

TGATACCTGCCGAATTGATGAAACTGAGCCTACGTTCAAA-ATGATTGGC
CGTGCAGAATACTTAATGGTATTATTTAATTAATGCTTTTAGGACATATA
TTTTTATAAA-AACTCACAACAGTTATT-TACAAGCTAAAACCCATTAC-
AACCATACTTT--TTAGTTAAACCCCCCC--ACCCCCAT-AAACT-AACA
TTATGCCCGAATAGCTATTCACTTCTCGTCAAACCCCTAAATCCGAGACT
AAC-----------------------------------------------
--------------------------------------------------
--------------------------------------------------
-----------------------
>DL;tortuga_de_mar_verde

TGATACCTGCCGATTCAGTGAAACTGGACTTACGTTTAAATATGATTGGC
CGTGCAAACTGATTAATGGTATTATTAAGTTAATGCTTATAAGACATAGA
ATTTCACAATTAAACCTAAACAATGATC-TACAACCTAA--CTCATTATT
AACTGTACTTT--TTAGCTAAACCCCCCT--ACCCCCGTTAAAGTCAACA
CCAGCCCGCTATAGCCATTTACTTCTCGCCAAACCCCTAAATCCGAGA--
--------------------------------------------------
--------------------------------------------------
--------------------------------------------------
-----------------------
>DL;Caballo_árabe

-------------------------GGACTTAAATTGAACGTTATCCCTC
CGCATAAGCAACCATAAGGTGTTATTCAGTTCATGGTAGCAGGACATAGG
AAACAAGTGCACCTGTGCACCTGTGCACCTGCCCGCGCAG-C-AAGCAAG
TAATATAGCTTTCTTAATCAAACCCCCCT-ACCCCCCATTAAACT-----
CCACATATGTACATTCAACACAATCTTGCCAAACCCCAAAAACAA-----
--------------------------------------------------
--------------------------------------------------
--------------------------------------------------
-----------------------
>DL;Cerdo

--------------------------------------------------
--------------------------------------------------
-------TACACGTGCGTACACGTGCG--TACACGCGCAT-ATAAGCAGG
TAA-ATTATTAGCTCATTCAAACCCCCCTTACCCCCCATTAAACT-----
TATGCTCTACACACCCTATAACGCCTTGCCAAACCCCAAAAACAAAGCA-
--------------------------------------------------
--------------------------------------------------
--------------------------------------------------
-----------------------
