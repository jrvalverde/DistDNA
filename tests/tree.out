Distance Matrix
---------------
Uncorrected for Multiple Substitutions
Using base positions 123 in the codon
Gap weighting is 0.000000

	       1	       2	       3	       4	       5	       6	       7	       8	       9	      10	      11	      12	      13	      14
	  0.00	 16.60	 39.75	 42.50	 24.61	 14.34	 14.73	 14.34	 14.34	 60.66	 70.86	 71.51	 69.05	 70.87		DL;Chimpancé 1
		  0.00	 39.45	 46.58	 24.08	 24.16	 20.82	 20.07	 20.07	 62.60	 76.00	 74.42	 70.24	 68.75		DL;Gorila 2
			  0.00	 35.40	 34.64	 44.98	 36.33	 35.99	 35.99	 62.10	 66.06	 63.58	 67.72	 66.15		DL;Macaco_japonés 3
				  0.00	 40.00	 43.12	 42.50	 43.75	 43.12	 61.86	 67.32	 63.58	 65.31	 66.39		DL;Tití 4
					  0.00	 24.61	 24.08	 23.56	 24.08	 63.41	 71.43	 71.51	 67.86	 65.62		DL;Orangután_de_Sumatra 5
						  0.00	 19.29	 19.58	 20.18	 63.11	 71.43	 72.09	 69.05	 70.08		DL;H.sapiens_neanderthalensis 6
							  0.00	  0.87	  0.87	 61.48	 71.43	 72.09	 69.64	 69.29		DL;Aborigen_australiano 7
								  0.00	  0.58	 63.11	 72.00	 72.67	 69.64	 68.50		DL;H.sapiens_Indonesio 8
									  0.00	 62.30	 71.43	 72.67	 70.83	 70.08		DL;H.sapiens_sapiens 9
										  0.00	 59.32	 60.00	 61.34	 60.68		DL;Geospiza_magnirostris-ave 10
											  0.00	 19.92	 48.53	 46.03		DL;Caretta_caretta-tortuga 11
												  0.00	 45.89	 45.67		DL;tortuga_de_mar_verde 12
													  0.00	 27.56		DL;Caballo_árabe 13
														  0.00		DL;Cerdo 14
